### PR TITLE
Main menu implementation

### PR DESCRIPTION
**Pull request by FerchusGames**

### CODE REVIEW

File: Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs [hunk 1]
```diff
@@ -228,7 +228,7 @@ private void CheckForControllerTypeChange()
         private void ChangeControllerType(ControllerType controllerType)
         {
             _currentControllerType = controllerType;
-            Debug.Log($"ControllerType: {_currentControllerType}");
+             Debug.Log($"ControllerType: {_currentControllerType}");
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs:231]: WTF

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/UIColors.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/UI/UIColors.cs:15]: What is this?

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs (hunk 1 above)
Comment (FerchusGames) [reply Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputEventManager.cs:231]: xd

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/UIColors.cs (hunk 2 above)
Comment (FerchusGames) [reply Where_Is_My_Wife/Assets/_Main/Scripts/UI/UIColors.cs:15]: Unused class I planned to store the UI colors like selected button color etc. Not needed so will be deleting it.

**Review by Horothenic — APPROVED** [review]

**Pull request merged**